CONTRIBUTING.md

```diff
@@ -1,4 +1,49 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
 ## Contributing to this Project
 
-OpenSearch is a community project that is built and maintained by people just like **you**.
+    OpenSearch is a community project that is built and maintained by people just like **you**.
 [This document](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) explains how you can contribute to this and related projects.
+The brief summary below outlines contribution guidelines specific to the OpenSearch-Security Repo.
+
+## Triaging
+
+OpenSearch-Security seeks to promote an inclusive and engaged community of contributors. In order to facilitate this, weekly triage meetings are open-to-all and attendance is encouraged for anyone who hopes to contribute, discuss an issue, or learn more about the project.
+
+FAQ about OpenSearch-Security Triage Meetings:
+
+### Do I need to attend for my issue to be addressed/triaged?
+
+Attendance is not required for your issue to be triaged or addressed. All new issues are triaged weekly.
+
+### How do I join the Backlog & Triage meeting?
+  
+Meetings are hosted regularly at 3 PM Eastern Time (Noon Pacific Time) and can be joined via the links posted on the [Upcoming Events](https://opensearch.org/events) webpage.
+
+After joining the Zoom meeting, you can enable your video / voice to join the discussion.  If you do not have a webcam or microphone available, you can still join in via the text chat.
+    
+If you have an issue you'd like to bring forth please consider getting a link to the issue so it can be presented to everyone in the meeting.
+
+### Is there an agenda for each week? 
+
+Meetings are lightly structured such that new issues are reviewed first, then the sprint backlog, and finally the backlog. There is no specific ordering within each category. 
```

Comment:
Thoughts on making these sections links to the filters in GH to get to those issues?